nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -351,89 +129,13 @@ def forward(self, x, *args):
     slice_list.append(slice_obj)
     return SliceMoudle(tuple(slice_list))
 
+def tupleunpack_python(node, speedup):
+    # Note: tuple unpack should only exists at the
+    # the end of the model, and is no need to replace/propagate mask
+    return None
```

Comment:
Is `None` specially handled by the caller? Seems all other similar functions return callables.
If it should never happen, then raise an exception.

Reply:
Yes, the returned value 'None' can be handled.

Reply:
should I declare the return value types of all these functions, such as `def tupleunpack_python(_node, _speedup) -> Optional[Callable]:`?